seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -872,7 +872,7 @@
         "capitalizeString(\"hello!\");"
       ],
       "tests": [
-        "assert(code.match(/import\\s+\\{\\s?capitalizeString\\s?\\}\\s+from\\s+\"string_functions\"/ig)"
+        "assert(code.match(/import\\s*\\{\\s*capitalizeString\\s*\\}\\s*from\\s*\\\"string_functions\\\"/ig), 'message: Import <code>capitalizeString</code> function from the <code>\"string_functions\"</code> file');"
```

Comment:
Was this added accidentally? Somewhat related https://github.com/freeCodeCamp/freeCodeCamp/pull/12745
Name change LGTM 👍 Should the file name also be changed?